### PR TITLE
Another round of fixes

### LINKED ISSUE #207: Modify gitter sidecar button to match our styles

### LINKED ISSUE #208: Posts by category is not working at all

### PR DESCRIPTION
**Pull request by XVincentX**
Connect #208 
Closes #208 

Connect #207 
Closes #207 

image

### CODE REVIEW

File: _posts/2018-02-02-amazon-api-gateway-compare-express-gateway.markdown [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: 'Developer Spotlight: Compare Amazon API Gateway vs Express Gateway'
 date: 2018-02-02 00:00:00 Z
 categories:
-- comparision
+- comparison
```
Comment (altsang) [inline _posts/2018-02-02-amazon-api-gateway-compare-express-gateway.markdown]: I think we use plural like `announcements`, just to keep consistent - `comparisons`?

File: _posts/2018-02-21-getting-started-with-environment-variables-and-express-gateway.markdown [hunk 2]
```diff
@@ -76,4 +76,4 @@ config file.
 If you’re ready to get started with Environment Variables in Express Gateway with this easy-to-follow How-To Video:
 
 <iframe src="https://player.vimeo.com/video/256160092" width="640" height="360" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
-<p><a href="https://vimeo.com/256160092">How To Use Environment Variables In Express Gateway</a> from <a href="https://vimeo.com/altsang">Al Tsang</a> on <a href="https://vimeo.com">Vimeo</a>.</p>
\ No newline at end of file
+<p><a href="https://vimeo.com/256160092">How To Use Environment Variables In Express Gateway</a> from <a href="https://vimeo.com/altsang">Al Tsang</a> on <a href="https://vimeo.com">Vimeo</a>.</p>
```
Comment (altsang) [inline _posts/2018-02-21-getting-started-with-environment-variables-and-express-gateway.markdown]: I think this should be `from Vincenzo Chianese`, not `Al Tsang`?

**Review by altsang — CHANGES_REQUESTED** [review]

File: _posts/2018-02-21-getting-started-with-environment-variables-and-express-gateway.markdown (hunk 2 above)
Comment (XVincentX) [reply _posts/2018-02-21-getting-started-with-environment-variables-and-express-gateway.markdown]: In theory yes; in practice I do not have a Vimeo account, so there's no link I could put.

How do you want me to handle this?

Comment (altsang) [reply _posts/2018-02-21-getting-started-with-environment-variables-and-express-gateway.markdown]: no it's ok, I'm just the poster, the text can just be changed, all of our EG and LB videos are under my paid account

i.e. 
`from Vincenzo Chianese on <a href="https://vimeo.com/altsang">Vimeo</a>`

**Pull request merged**